### PR TITLE
Add EL9 to packit config

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -28,12 +28,18 @@ jobs:
...
     targets:
-      centos-stream-8:
+      rhel-8:
         additional_modules: "katello:el8,foreman-devel:el8"
         additional_repos:
           - http://yum.theforeman.org/releases/nightly/el8/x86_64/
           - http://yum.theforeman.org/plugins/nightly/el8/x86_64/
           - http://yum.theforeman.org/katello/nightly/katello/el8/x86_64/
+      rhel-9:
+        additional_modules: "katello:el9,foreman-devel:el9"
```
Comment (maximiliankolb) [inline .packit.yaml:38]: On EL9, you don't have to enable DNF modules: https://docs.theforeman.org/nightly/Installing_Server/index-katello.html#repositories-el-9

packit log:

````
Error: Problems in request:
missing groups or modules: katello:el9
WARNING: DNF4 command failed, retrying, attempt #2, sleeping 10s
No matches found for the following disable plugin patterns: local, spacewalk, versionlock
````

Comment (nadjaheitmann) [reply .packit.yaml]: Packit works if I add the Zeitwerk fix to it... What shall we do? 1) I can remove the Zeitwerk commits and we can merge this PR as is or 2) I can add the commit from this PR to the Zeitwerk PR #141 .

**Review by maximiliankolb — APPROVED** [review]
Thanks Nadja, diff in README and packit LGTM & I _do_ like a green GHA :partying_face: 

Did someone already review the other commits, perhaps internally or on a different PR? If so, please rebase/squash where necessary and merge. If now, maybe ping Bastian.

**Comment by nadjaheitmann** [comment]
> Did someone already review the other commits, perhaps internally or on a different PR? If so, please rebase/squash where necessary and merge. If now, maybe ping Bastian.

@ofedoren actually had a quick look at my Zeitwerk change. But I dod some more changes to it since then :)

**Review by m-bucher — APPROVED** [review]
Looks good to me. If the pipeline is green, feel free to merge it.

**Pull request merged**